npm-packages.md

```diff
@@ -93,11 +93,17 @@ The CommonJS version should be referenced by the `main` field, and the TypeScrip
 
 #### Continuous delivery
 
-Use [Semantic Release](https://github.com/semantic-release/semantic-release) in a GitHub action.
+Prefer continuous delivery from GitHub, using either [Semantic Release](https://github.com/semantic-release/semantic-release) (see below) or [Changesets](https://github.com/changesets/changesets) in a GitHub action.
```

Comment:
Not sure if this is the right time to go into this, but do we want to suggest using Changesets and avoid mentioning Semantic Release? It feels like the happy path right now, although maybe yet not totally paved 😆